vitest.shared.mts

```diff
@@ -1,11 +1,21 @@
 import { resolve, dirname } from 'node:path';
 import { fileURLToPath } from 'node:url';
 import { defineConfig } from 'vitest/config';
+import babel from 'vite-plugin-babel';
 
 const CURRENT_DIR = dirname(fileURLToPath(import.meta.url));
 const WORKSPACE_ROOT = resolve(CURRENT_DIR, './');
 
 export default defineConfig({
+  plugins: [
+    babel({
+      babelConfig: {
+        plugins: ['@mui/internal-babel-plugin-display-name'],
+        babelrc: false,
+        configFile: false,
+      },
+    }),
+  ],
```

Comment:
Thanks for looking into this :+1: 

It might be nice to import our actual `babel.config.js` as the config there, we have a few plugins that modify the code so it would be nice to test under the same conditions that our code actually runs in production.

Reply:
Yeah, but it seems we had decided not to run our babel config in vitest, it would probably require a lot of changes too https://github.com/mui/mui-x/pull/17845#issuecomment-2890703266